.github/workflows/zowe-cli.yml

```diff
@@ -136,26 +142,78 @@ jobs:
       run: ./__tests__/__resources__/daemon_instances/zowe
 
     - name: Integration Tests (Native)
-      id: integration-native
       if: ${{ always() && steps.build.outcome == 'success' && !(github.event.inputs.test-type == 'binary' || github.event_name == 'push') }}
       run: npm run test:integration
 
     - name: Integration Tests (Daemon)
-      id: integration-daemon
       if: ${{ always() && steps.build.outcome == 'success' && (github.event.inputs.test-type == 'binary' || github.event_name == 'push') }}
       run: npm run test:integration -- --runInBand
 
     - name: Archive Results
-      id: upload
       if: ${{ always() && steps.build.outcome == 'success' }}
       uses: actions/upload-artifact@v2
       with:
         name: ${{ matrix.os }}-${{ matrix.node-version }}-results
         path: __tests__/__results__/
 
     - name: Upload Results to Codecov
-      id: upload-codecov
       if: ${{ always() && steps.build.outcome == 'success' }}
       uses: codecov/codecov-action@v3
       with:
         env_vars: OS,NODE
+
+    - name: Configure Sonar Scan
+      if: ${{ always() && steps.build.outcome == 'success' && matrix.os == 'ubuntu-latest' && matrix.node-version == '16.x' }}
+      uses: zowe-actions/octorelease-script@update-for-cli
+      with:
+        script: sonarConfig
+
+    - name: SonarCloud Scan
+      if: ${{ always() && steps.build.outcome == 'success' && matrix.os == 'ubuntu-latest' && matrix.node-version == '16.x' }}
+      uses: sonarsource/sonarcloud-github-action@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONARCLOUD_TOKEN }}
+
+  release:
+    if: github.event_name == 'push'
+    needs: test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        persist-credentials: false
+        ref: ${{ github.ref }}
+
+    - name: Use Node.js LTS
+      uses: actions/setup-node@v2
+      with:
+        node-version: 'lts/*'
+
+    - name: Install Dependencies
+      run: npm ci
+
+    - name: Update Dependencies
+      uses: zowe-actions/octorelease-script@update-for-cli
+      env:
+        GIT_COMMITTER_NAME: ${{ secrets.ZOWE_ROBOT_USER }}
+        GIT_COMMITTER_EMAIL: ${{ secrets.ZOWE_ROBOT_EMAIL }}
+        NPM_RESOLUTIONS: ${{ needs.test.outputs.npm-resolutions }}
```

Comment:
Offline Q: Does this mean that the `zowe-actions/octorelease-script:npmUpdate` sets the `output.results` to an object containing the `npm-resolutions` ?

(Just trying to understand. Sorry for all the dumb questions. I don't mind posting an update later)

Reply:
That's right, not a dumb question 🙂 The relevant changes are in this PR: https://github.com/zowe-actions/octorelease-script/pull/6